src/eradius_lib.erl

```diff
@@ -462,8 +460,8 @@ do_scramble(_SharedSecret, _B, << >>, CipherText) ->
 
 -spec generate_salt() -> salt().
 generate_salt() ->
-    Salt1 = crypto:rand_uniform(128, 256),
-    Salt2 = crypto:rand_uniform(0, 256),
+    Salt1 = 127 + rand:uniform(128),
+    Salt2 = rand:uniform(256) - 1,
     <<Salt1, Salt2>>.
```

Comment:
what about:

```
<<Salt1, Salt2>> = crypto:strong_rand_bytes(2),
<<Salt1 bor 16#80, Salt2>>.
```

We are already using crypto:strong_rand_bytes/1 for the authenticator -- and don't handle the low_entropy error there as well.